community-content/tf_agents_bandits_movie_recommendation_with_kfp_and_vertex_sdk/mlops_pipeline_tf_agents_bandits_movie_recommendation/mlops_pipeline_tf_agents_bandits_movie_recommendation.ipynb

```diff
@@ -966,6 +989,10 @@
         "        confidence intervals of the Trainer.\n",
         "    \"\"\"\n",
         "    # Run the Generator component.\n",
+        "    generate_op = create_component_from_func(\n",
```

Comment:
I'm not sure we should go back from loading the reusable components to generating them for every pipeline run.
Perhaps there is some confusion regarding the way the components can be updated?

Reply:
One way to update strict component references is to make change to the components, commit the change, use `git log` to see the commit SHA, then replace the SHA in the URL with the updated SHA.